tools/tf2openapi/generator/generate.go

```diff
@@ -4,7 +4,12 @@ import (
 	pb "github.com/kubeflow/kfserving/tools/tf2openapi/generated/protobuf"
 )
 
-func GenerateOpenAPI(model *pb.SavedModel) (string, error) {
+const defaultSigDefKey = "serving_default"
+
+func GenerateOpenAPI(model *pb.SavedModel, sigDefKey string) (string, error) {
```

Comment:
If this function signature gets too complicated, we may need to think about a builder pattern for the Generator.